spec/requests/posts_spec.rb

```diff
@@ -1,7 +1,44 @@
 require 'rails_helper'
 
 RSpec.describe 'Posts', type: :request do
-  describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+  describe 'GET /users/:user_id/posts' do
```

Comment:
- [ ] Great job working on this milestone. You should test the index and show actions and that should be reflected in your describe. See below.
```
describe 'GET /index' do
  ...
end
```